source/payment_flow/index.html.md.erb

```diff
@@ -320,19 +320,19 @@ more information about how you can integrate your service with GOV.UK Pay.
 
 ### Resuming an incomplete payment
 
-You can send your user back to GOV.UK Pay if they arrive at your `return_url` before they've finished their payment. For example, your user selected their web browser’s **Back** button while making a payment, then selected **Forward**.
+You can send your user back to GOV.UK Pay if they return to your service before they've finished their payment.
 
-You can only resume a payment if the `status` is one of the following:
+[Get information about the payment](/reporting/#get-information-about-a-single-payment) to check the payment's `status`.  the payment is incomplete if the `status` is one of the following:
```

Comment:
Need a capital letter at the beginning of that last sentence!

Reply:
Oh dear - thanks for spotting!